aeon_core_v1_2.py

```
@@ -0,0 +1,294 @@
+        with open(self.graph_path, 'w') as f:
+            json.dump(data, f)
+
+    def load_state(self):
+        """Load graph from JSON if exists."""
+        if os.path.exists(self.graph_path):
+            with open(self.graph_path, 'r') as f:
+                data = json.load(f)
+                self.G = nx.node_link_graph(data["graph"])
+                self.metadata = data.get("metadata", {})
+
+    def add_cme_event(self, state: np.ndarray, intent: np.ndarray, outcome: str,
```

Ruff (0.12.2)

Unused method argument: `outcome`

(ARG002)